openslides/motions/static/templates/motions/motion-detail.html

```diff
@@ -370,3 +370,10 @@ <h3 translate>Version history</h3>
 
   </div>
 </div>
+
+<div class="details">
+    <h3>Motion Comments</h3>
+    <p ng-repeat="field in commentsFields">
+        {{ field.name }}: {{ motion.comments[$index] }}
```

Comment:
Here we need a check: show privte field names (and values) only if user has right permission.